src/Configurations/PageRulesActions.php

```diff
@@ -54,29 +46,23 @@ public function setBypassCacheOnCookie(bool $value)
             throw new ConfigurationsException("Invalid cookie string.");
         }
 
-        $object = new \stdClass();
-        $object->id = "bypass_cache_on_cookie";
-        $object->value = $value;
```

Comment:
This is a `bool` but get passed in raw unlike the on/off conversation, is this correct?

Reply:
There's a slight inaccuracy I'll adjust here in a separate PR, in essence the value tied to Bypass Cache on Cookie (or Cache on Cookie) for that matter should be a string.